cranelift/codegen/src/isa/aarch64/mod.rs

```diff
@@ -109,6 +110,31 @@ impl MachBackend for AArch64Backend {
         // opposite of x86).
         IntCC::UnsignedLessThan
     }
+
+    #[cfg(feature = "unwind")]
+    fn emit_unwind_info(
+        &self,
+        result: &MachCompileResult,
+        kind: crate::machinst::UnwindInfoKind,
+    ) -> CodegenResult<Option<crate::isa::unwind::UnwindInfo>> {
+        use crate::isa::unwind::UnwindInfo;
+        use crate::machinst::UnwindInfoKind;
+        Ok(match (result.unwind_info.as_ref(), kind) {
+            (Some(info), UnwindInfoKind::SystemV) => {
+                inst::unwind::systemv::create_unwind_info(info.clone())?.map(UnwindInfo::SystemV)
+            }
+            (Some(_info), UnwindInfoKind::Windows) => {
+                //TODO inst::unwind::winx64::create_unwind_info(info.clone())?.map(|u| UnwindInfo::WindowsX64(u))
```

Comment:
`winx64` left over from copy+paste?

Reply:
yes, but it is also a TODO to support windows fastcall abi

Reply:
OK, gotcha, maybe something like `TODO: support Windows unwind info on AArch64` then?